src/drivers/uavcan/actuators/esc.cpp

```diff
@@ -51,6 +51,7 @@ UavcanEscController::UavcanEscController(uavcan::INode &node) :
 	_uavcan_sub_status(node)
 {
 	_uavcan_pub_raw_cmd.setPriority(UAVCAN_COMMAND_TRANSFER_PRIORITY);
+	param_get(param_find("UAVCAN_ESC_LR"), &_uavcan_use_rate_limit);
```

Comment:
UAVCAN_ESC_RATELIMIT? or RL? maybe not LR?

Reply:
yeah, I will change the name so it's more intuitive